test/syntax.jl

```diff
@@ -546,7 +546,7 @@ for (str, tag) in Dict("" => :none, "\"" => :string, "#=" => :comment, "'" => :c
 end
 
 # meta nodes for optional positional arguments
-@test Meta.lower(Main, :(@inline f(p::Int=2) = 3)).args[1].code[end-1].args[3].inlineable
+@test Meta.lower(Main, :(@inline f(p::Int=2) = 3)).args[1].code[end-1].args[3].inlining_cost == 10
```

Comment:
Using the hardcoded `10` here is fragile against possible changes in our inlining heuristic implementation. How about using `Core.Compiler.inlineable` instead?